changelog/druntime.nogc-traceinfo.dd

```diff
@@ -0,0 +1,16 @@
+The default `Throwable.TraceInfo` generation now is `@nogc`. The way this works:
```

Comment:
```suggestion
The default `Throwable.TraceInfo` generation now is `@nogc`. 

The way this works:
```

Separate the description from the first line, which is the title.

Reply:
Done